### PR TITLE
Various mappings in net/minecraft/world/entity

### PR DESCRIPTION
**Pull request by PiTheGuy**

### CODE REVIEW

File: data/net/minecraft/world/entity/AgeableMob.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/AgeableMob.mapping:3]: This is correct, the other argument should probably be "otherParent"

File: data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping]: ``onPoiAcquistutionEvent`` -> ``onPoiAcquisitionEvent``

File: data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping:17]: The order is:
``memoryKey`` & ``memoryToAcquire`` i think.
Either way the second value is ``memoryToAcquire`` not the first, see the call to super in AcquirePoi constructor.

File: data/net/minecraft/world/entity/ai/behavior/GateBehavior.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/GateBehavior.mapping:31]: This should be in theory ``behaviour, level, owner, gameTime``

File: data/net/minecraft/world/entity/ai/behavior/GoToWantedItem.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/GoToWantedItem.mapping:3]: Arg 2 should be ``hasTarget``

File: data/net/minecraft/world/entity/ai/behavior/PlayTagWithOtherKids.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/PlayTagWithOtherKids.mapping:4]: arg 1 is entity or memoryHolder
arg 2 is interactionTarget

File: data/net/minecraft/world/entity/ai/behavior/RunIf.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/RunIf.mapping:8]: arg 1 is entryConditions

File: data/net/minecraft/world/entity/ai/behavior/RunSometimes.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/RunSometimes.mapping:6]: arg2 should be ``keepTicks``

File: data/net/minecraft/world/entity/ai/behavior/SetHiddenState.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetHiddenState.mapping:3]: ``stayHiddenSeconds`` or ``durationInSeconds`` ???
@alcatrazEscapee @ChampionAsh5357

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping:5]: arg 4 is "hasTarget"

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping:14]: arg 3 is "hasTarget"

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetAwayFrom.mapping:19]: arg 3 is "hasTarget"

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping:9]: Order of arguments are: ``level, villager, memoryPos``

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping:23]: Order of arguments are: ``level, memoryPos``

File: data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.mapping:17]: Order of arguments are: ``level, villager``

File: data/net/minecraft/world/entity/ai/behavior/ValidateNearbyPoi.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/ValidateNearbyPoi.mapping:7]: Order of arguments are: ``level, pos, sleeper``

File: data/net/minecraft/world/entity/ai/behavior/WorkAtComposter.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/ai/behavior/WorkAtComposter.mapping:9]: Order of arguments are: ``level, preState, pos, postState``

File: data/net/minecraft/world/entity/animal/Rabbit.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/animal/Rabbit.mapping]: ``item`` -> ``stack``

File: data/net/minecraft/world/entity/animal/TropicalFish.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/animal/TropicalFish.mapping:9]: Order of arguments are: ``type, level, spawnType, pos, random``

File: data/net/minecraft/world/entity/animal/Wolf.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/animal/Wolf.mapping:14]: Order of arguments are: ``type, level, spawnType, pos, random``

File: data/net/minecraft/world/entity/animal/goat/Goat.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/animal/goat/Goat.mapping]: ``screamingGoat`` -> ``isScreamingGoat``

File: data/net/minecraft/world/entity/monster/Skeleton.mapping
Comment (Lanse505) [inline data/net/minecraft/world/entity/monster/Skeleton.mapping]: ``freezeConverting`` -> ``isFrozen``

**Review by Lanse505 — CHANGES_REQUESTED** [review]
Overall looks good, some changes need to be done before it can be merged though.
There is a singular javadoc change, not too experienced with our javadocs policy so I'll defer it to @alcatrazEscapee @sciwhiz12 or @ChampionAsh5357

File: data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping
Comment (PiTheGuy) [reply data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping:17]: I'll add `memoryKey` as the name for the first parameter, but in Enigma it already shows `memoryToAcquire` as the second parameter, not the first.

File: data/net/minecraft/world/entity/AgeableMob.mapping
Comment (PiTheGuy) [reply data/net/minecraft/world/entity/AgeableMob.mapping:3]: Changed

File: data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping
Comment (PiTheGuy) [reply data/net/minecraft/world/entity/ai/behavior/AcquirePoi.mapping]: Fixed

**Comment by PiTheGuy** [comment]
All requested changes should have been applied. The only exception is the one waiting for a second opinion from members of the team.

File: data/net/minecraft/world/entity/ai/behavior/SetHiddenState.mapping
Comment (alcatrazEscapee) [reply data/net/minecraft/world/entity/ai/behavior/SetHiddenState.mapping:3]: Mojang field is `stayHiddenTicks`, the parameter is multiplied by 20 and assigned, I think `stayHiddenSeconds` makes sense here.

File: data/net/minecraft/world/entity/player/Inventory.mapping
Comment (alcatrazEscapee) [inline data/net/minecraft/world/entity/player/Inventory.mapping]:
```suggestion
		COMMENT @return a player armor item (as an {@code ItemStack}) contained in specified armor slot
```

Generally for formatting, we allow for `@return` and `@param`, any of the following styles:
 - `@return the foo`
 - `@return the foo. And another full sentence.`
 - `@return The Foo. And another full sentence.`

**Review by Lanse505 — APPROVED** [review]
Looks good to me

**Pull request merged**
